src/pages/Detail/DetailHotelFacility/FacilitiesIconWrap.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import styled from 'styled-components';
+import { flexSet } from '../../../styles/Mixins';
+
+export const FacilitiesIconWrap = ({ text }) => {
+  return (
+    <FacilityIconBox>
+      <img src={FACILITIES_LIST[`${text}`]} alt="parking" />
```

Comment:
```suggestion
      <img src={FACILITIES_LIST[text]} alt="parking" />
```
로 하면 동작 안해서 저렇게 처리하신건가요?

Reply:
아 동작을 합니다..! 감사합니다